source/Assets/Plugins/Didomi/Scripts/IOS/DDMEventType.cs

```diff
@@ -48,8 +48,9 @@ public enum DDMEventType
         DDMEventTypePreferencesClickSPICategoryAgree = 38,
         DDMEventTypePreferencesClickSPICategoryDisagree = 39,
         DDMEventTypePreferencesClickSPIPurposeSaveChoices = 40,
-        DDMEventTypeDCSSignatureReady = 41,
-        DDMEventTypeDCSSignatureError = 42,
+        DDMEventTypeDCSSignatureError = 41,
+        DDMEventTypeDCSSignatureReady = 42,
+        DDMEventTypeIntegrationError = 43,
```

Comment:
The order of values has to match the order in native `EventType.swift`:

```
    /// DCS Signature
    case dcsSignatureReady
    case dcsSignatureError
    
    /// Integrations
    case integrationError
```

Reply:
pushed